.fixtures.yml

```diff
@@ -4,3 +4,4 @@
 fixtures:
   forge_modules:
     systemd: "puppet/systemd"
+    stdlib: "puppetlabs/stdlib"
```

Comment:
Please use links to git repos instead:
```
fixtures:
  repositories:
    stdlib: https://github.com/puppetlabs/puppetlabs-stdlib.git
   systemd: https://github.com/voxpupuli/puppet-systemd.git
```

We test against the main branches of our dependencies to detect breaking changes before they are released. We use forge releases within our acceptance tests.